### ISSUE TITLE
Any plans for support of other OGC Protocols, especially WMTS?

### ISSUE THREAD

**Issue by cpaulik** [pmlrsg/GISportal#47]
Last week the [Copernicus Viewing Service](http://viewer.globalland.vgt.vito.be/viewer/) was announced and I discovered that they are using WMTS instead of WMS. Since we would like to integrate at least one of their products in our viewer it would be interesting to use their servers instead of hosting it ourselves. Are there any plans already to also support this protocol? If not then I might be able to work on that in a few weeks.

**Comment by ojsdude** [comment]
We currently have no plans to support this in the near future, it would obviously be something that would be a fantastic addition to the portal. If you do get chance to have a look at it then by all means that would be fantastic. If you need a pointer for anything then continue on this thread and I will get back to you.

**Comment by ojsdude** [comment]
Hey Christoph,

I just wanted to ask if you have had a look at implementing WMTS support at all?

Thanks,

OJ

**Comment by cpaulik** [comment]
No, I didn't find the time for it. It is still on my todo list but I doubt that I'll get to it in the next months.

**Comment by ojsdude** [comment]
Okay, not to worry, I will start to have a look at it now and I will let you know how it goes
